examples/ieee14bus/buildexample.jl

```diff
@@ -0,0 +1,46 @@
+using PowerDynamics: SlackAlgebraic, FourthOrderEq, VoltageDependentLoad, PiModelLine, StaticLine, Transformer, PowerGrid, write_powergrid, Json
+using OrderedCollections: OrderedDict
+
+# Data Source: Kodsi, S. K. M., & Canizares, C. A. (2003). Modeling and simulation of IEEE 14-bus system with FACTS controllers. University of Waterloo, Canada, Tech. Rep.
+
+buses=OrderedDict(
+    "bus1"=> FourthOrderEq(T_d_dash=7.4, D=2, X_d=0.8979, X_q=0.646, Ω=50, X_d_dash=0.2995, T_q_dash=0.1, X_q_dash=0.646, P=2.32, H=5.148, E_f=1),
+    "bus2"=> SlackAlgebraic(U=1),
+    "bus3"=> FourthOrderEq(T_d_dash=6.1, D=2, X_d=1.05, X_q=0.98, Ω=50, X_d_dash=0.185, T_q_dash=0.4, X_q_dash=0.36, P=-0.942, H=6.54, E_f= 1),
+    "bus4"=> VoltageDependentLoad(P=-0.478, Q=-0.0, U=1.0, A=0.0, B=0.0),
```

Comment:
You are using `VoltageDependentLoad`'s here but the factors are all `0`. Is that for a reason?

Reply:
Not really. I thought it might be easier for users to play around with different load models by just changing parameters instead of replacing node models. But you're right, it is a bit confusing right now. I think I will either go back to the `PQAlgebraic` or change the parameters in the `VoltageDependentLoad` to make it actually voltage dependent.